### PR TITLE
Schema Migrator Go Upgrade

### PR DESCRIPTION
**Pull request by ralikio**
**Description**

Changes proposed in this pull request:

- schema migrator go upgrade,
- old toolbox removal,
- migration from dep to go mod for compatibility with generic makefiles (`go test ./...`)

**Related issue(s)**

### CODE REVIEW

**Review by tillknuesting — APPROVED** [review]

**Review by wozniakjan — COMMENTED** [review]
I think we should drop the `sed`, otherwise looking good

File: components/schema-migrator/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,14 @@
+FROM golang:1.18.0-alpine3.15 AS build
+
+WORKDIR /migrate
+
+RUN apk --no-cache add --update git make
+RUN git clone https://github.com/golang-migrate/migrate.git .
+
+RUN sed -i 's/^go .*/go 1.18/g' go.mod && go mod tidy && make build
```
[the remark this answers is not in the document]
Comment (wozniakjan) [reply components/schema-migrator/Dockerfile:8]: even easier to check with go natively :)
```
$ go version ./main
./main: go1.18
```

[the remark this answers is not in the document]
Comment (ralikio) [reply components/schema-migrator/Dockerfile:8]: Thank you! Corrected.

**Review by wozniakjan — APPROVED** [review]
/approve
/lgtm

**Pull request merged**